torch/ao/quantization/fx/convert.py

```diff
@@ -161,8 +162,28 @@ def _replace_observer_with_quantize_dequantize_node(
         # TODO(future PR): switch compute_dtype to is_dynamic
         # dynamic quantization
         node_type = "call_function"
+        is_dynamic = True
         if is_decomposed:
-            raise NotImplementedError("decomposed quantize_per_tensor_dynamic op not implemented yet")
+            # TODO: add quant_min/quant_max for PlaceholderObserver
+            # quant_min = activation_post_process.quant_min  # type: ignore[attr-defined]
+            # quant_max = activation_post_process.quant_max  # type: ignore[attr-defined]
+            dtype = to_underlying_dtype(dtype)
+            if dtype == torch.uint8:
```

Comment:
can this be reused with the observer logic in `choose_qparams`?  Ideally there is only one place where this stuff is defined

Reply:
oh this is temporary workaround when the PlacholderObserver doesn't have quant_min/quant_max defined, I think ideally we should just enforce quant_min/quant_max there, choose_qparams takes quant_min/quant_max as argument so we don't have logic to derive quant_min/quant_max from dtype in choose_qparams

Reply:
I can add a follow up PR to make sure PlaceholderObserver has this defined and remove this hack

Reply:
I think I rate this type of technical debt (hardcoding numbers that are used in mathematical calculations) as "best to never land in the codebase unless it's the final form", it's never temporary once you land it :)  This is riskier than other types of tech debt.

maybe it can be in this PR or in a PR before this one in a stack?

Reply:
yeah sure, I can do that